packages/webapp/src/pages/delegates/index.tsx

```diff
@@ -22,80 +33,128 @@ interface Props {
 const MEMBERS_PAGE_SIZE = 18;
```

Comment:
I think we only need fetch 1 here, not 18.

Reply:
This is fetching "all" members, so not just one